lib/plan-builder-generated.js

```diff
@@ -8279,7 +8280,8 @@ SpellExpr: SpellExpr,
 SqlExpr: SqlExpr,
 XdmpExpr: XdmpExpr,
 XsExpr: XsExpr,
-  Plan:       PlanPlan,
-  AccessPlan: PlanAccessPlan,
-  PlanColumn: PlanColumn
+Plan:       PlanPlan,
+AccessPlan: PlanAccessPlan,
+PlanColumn: PlanColumn,
+PlanExprCol: PlanExprCol
```

Comment:
Let's just revert all of the changes to plan-builder-generated.js

They're just indentation, so it's better to avoid adding an entry on the version stream for this module.